### ISSUE TITLE
[a11y]: ProductiveCard (story only update): graph should contain alt text

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-products#5862]
### Package

@carbon/ibm-products

### Browser

Firefox

### Operating System

Windows

### Package version

Latest

### Automated testing tool and ruleset

Manual

### Assistive technology

JAWS

### Description

The graph from the productive card stories should contain some kind of alt text. Notes from Raghu:

Visually there is a PI Chart (raph ) with colors red, gray and green partitions, which visually indicate some info . Text alternative is not provided for the color partitions hence screen reader users will not come to know about the graph information.

The info presented through the PI Chart (Graph ) must be conveyed to Screen reader users  by providing appropriate Text alternatives .

### WCAG 2.1 Violation

1.1.1   Non-Text Content

### Reproduction/example

Visible from productive card stories when using screen reader

### Steps to reproduce

See productive card stories with screen reader, there's no alternative way to understand the graph.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems
